packages/core-state/src/defaults.ts

```diff
@@ -4,6 +4,6 @@ export const defaults = {
         maxLastTransactionIds: 10000,
     },
     walletSync: {
-        enabled: false,
+        enabled: process.env.CORE_WALLET_SYNC_ENABLED === "true",
```

Comment:
Follow the convention of the rest of core https://github.com/ArkEcosystem/core/blob/develop/packages/core-api/src/defaults.ts#L4

Reply:
`enabled: !!process.env.CORE_WALLET_SYNC_ENABLED`?